src/sdd/ForwardTranslator.cpp

```diff
@@ -459,6 +460,7 @@ namespace sdd {
     m_ignoreTypes.push_back(model::ModelObjectList::iddObjectType());
     m_ignoreTypes.push_back(model::Node::iddObjectType());
     m_ignoreTypes.push_back(model::OutputControlReportingTolerances::iddObjectType());
+    m_ignoreTypes.push_back(model::OutputControlResilienceSummaries::iddObjectType());
```

Comment:
Unrelated to this PR, but... should other OutputControl objects be in this list?

Reply:
Yes. OutputSchedules too for eg.

I don't know who uses SDD these days to be honest, and I'll happily just ignore this until someone complains about the warnings